tests/wicked/before_test.pm

```diff
@@ -80,6 +88,10 @@ sub run {
             assert_script_run('./autogen.sh ',       timeout => 600);
             assert_script_run('make ; make install', timeout => 600);
         }
+        if (check_var('WICKED', 'ipv6')) {
+            zypper_ar('http://download.suse.de/ibs/home:/wicked-maintainers:/openQA/SLE_15_SP2/', name => 'wicked_maintainers', no_gpg_check => 1, priority => 60);
```

Comment:
currently this package have a **single** repository , so if I will make it "flexible" by doing this 
```suggestion
            zypper_ar('http://download.suse.de/ibs/home:/wicked-maintainers:/openQA/get_var('VERSION')/', name => 'wicked_maintainers', no_gpg_check => 1, priority => 60);
```

I would need also add : 
`die "Unsupported version" unless get_var('VERSION') eq 'SLE15-SP2'` :) 
if you think this will be improvement I can do this but as for me it is not

Reply:
I'm ok to leave it like this for now. But we have to keep in mind that we can't execute this one in O3 for example.

Reply:
yeah sure but just keep ion mind that currently we executing **only** wicked_basic on O3 :)